paddle/fluid/inference/tests/api/analyzer_vis_tester.cc

```diff
@@ -61,8 +61,6 @@ void SetConfig(AnalysisConfig *cfg) {
   cfg->ir_passes.push_back("fc_gru_fuse_pass");
 #ifdef PADDLE_WITH_MKLDNN
   cfg->_use_mkldnn = true;
-  // disable mkldnn fuse since it should have some bugs
```

Comment:
Will you add back?

Reply:
Now that the MKL-DNN is properly called the test no longer fails when this pass is used. There is no need for disabling it.